tests/main/interfaces-many-core-provided/task.yaml

```diff
@@ -27,6 +27,14 @@ environment:
     CONSUMER_SNAP: test-snapd-policy-app-consumer
 
 prepare: |
+    # We skip the test in trusty butcause it fails with the following error
+    # since was added private /dev/shm support to shared-memory interface
+    # shared-memory plug with "private: true" cannot be connected if "/dev/shm"
+    # is a symlink)
```

Comment:
```suggestion
    # We skip the test in trusty because it fails with the following error
    # since adding private /dev/shm support to shared-memory interface:
    # shared-memory plug with "private: true" cannot be connected if "/dev/shm"
    # is a symlink)
```